Doc/library/typing.rst

```diff
@@ -2773,6 +2773,10 @@ Introspection helpers
    .. versionchanged:: 3.9
       Added ``include_extras`` parameter as part of :pep:`593`.
 
+   .. versionchanged:: 3.10
+      ``__annotations__`` of a class does not contain anymore the annotations
```

Comment:
As a maintainer of several runtime libraries that work with annotaions, I would expect to see it here, because it is the main entrypoint for runtime annotations.

Yes, technically it was not changed, but `.__annotations__` was.

Reply:
Maybe the note here should say what changed in the observable behavior of `get_type_hints()` instead of talking about `__annotations__`.
```suggestion
      Calling ``get_type_hints()`` on a class no longer returns the annotations
```

Reply:
@JelleZijlstra that's a great compromise. I've modified the entry.